app/src/main/java/com/mikhailovskii/weatherandroid/util/WeatherCalculations.kt

```diff
@@ -42,4 +24,13 @@ fun getDateFromSeconds(seconds: Long): String {
     val date = Date(seconds * 1000)
 
     return simpleDateFormat.format(date)
-}
\ No newline at end of file
+}
+
+class WeatherUtils {
```

Comment:
@SergeiMikhailovskii looks like `getWindDirection()` can also be a part of `WeatherUtils`, doesn't it?